litmus-portal/graphql-server/pkg/cache/cache.go

```diff
@@ -0,0 +1,22 @@
+package cache
```

Comment:
Put the cache.go in pkg/utils directory. We are planing to move some other files to utils directory also.